package.json

```diff
@@ -61,17 +61,20 @@
     "@rollup/plugin-json": "^4.0.0",
     "@rollup/plugin-node-resolve": "^9.0.0",
     "@rollup/plugin-replace": "^2.0.0",
+    "@types/react": "^16.9.53",
+    "@types/react-dom": "^16.9.8",
```

Comment:
I guess these are kinda like peer deps, right? In that case, I’m assuming using the `16.0.0` versions will reduce noice in the future

Reply:
Added as peer dependencies, for testing I'd recommend staying with React 16.
It will reduce noise and hopefully ensure a smooth transition to react 17 ([coming soon](https://reactjs.org/blog/2020/08/10/react-v17-rc.html))